src/ontology/uberon-edit.obo

```diff
@@ -214638,13 +214640,14 @@ replaced_by: UBERON:0001675
 
 [Term]
 id: UBERON:3011048
-name: genital system
-def: "Anatomical system that has as its parts the organs concerned with reproduction, especially the external genital organs." [https://www.merriam-webster.com/dictionary/genitalia, XAO:0000142]
+name: obsolete genital system
+def: "OBSOLETE Anatomical system that has as its parts the organs concerned with reproduction, especially the external genital organs." [https://www.merriam-webster.com/dictionary/genitalia, XAO:0000142]
+comment: https://github.com/obophenotype/uberon/issues/2493
```

Comment:
in Mondo, we use seeAlso for the URL. 

Is there any documentation anywhere about the obsoletion workflow for Uberon?

Reply:
Let's make the workflows 100% identical. Are there differences other than prepending the definition string? Let's just make a universal decision across our ontologies yes/no to prepending definition strings.

(I don't think the comment comment is related to the obsoletion workflow per se, but in general Uberon should use largely the same APs as Mondo, and the same way of relating a term to an issue)

Reply:
In GO we use [IAO:0000233](http://purl.obolibrary.org/obo/IAO_0000233)

Reply:
should we use [IAO:0000233](http://purl.obolibrary.org/obo/IAO_0000233) term tracker item in Mondo too? I can write up some documentation on obsoletion workflows. Should I add it to OBO Academy? We want that to be the definitive source, right?